tripy/nvtripy/backend/api/shape_bounds.py

```diff
@@ -59,3 +59,28 @@ def decode_shape_bounds(shape_bounds_dict):
         opt=tuple(shape_bounds_dict["opt"]),
         max=tuple(shape_bounds_dict["max"]),
     )
+
+
+@dataclass
+class ValueBounds:
```

Comment:
This seems basically identical to `ShapeBounds`. Should we just generalize that class to `Bounds`?